shared/menubar/index.render.desktop.js

```diff
@@ -116,16 +116,16 @@ const Row = props => {
   return (
     <div style={containerStyle} onClick={props.onClick}>
       <div style={{...globalStyles.clickable, marginRight: 2, ...props.iconStyle}}/>
-      <Text type='Body' link small key={props.key} style={{marginTop: 4, overflowWrap: 'break-word', flex: 1, ...props.textStyle}}>{props.text}</Text>
+      <Text type='Body' link small style={{marginTop: 4, overflowWrap: 'break-word', flex: 1, ...props.textStyle}}>{props.text}</Text>
       {props.children}
     </div>
   )
 }
 
 const FolderRow = props => {
-  const divider = <span>,<wbr/></span> // word break on the commas
+  const divider = idx => <span key={'div' + idx}>,<wbr/></span> // word break on the commas
```

Comment:
we have to use the dividerFn to get keyd dividers ,else all the dividers will get thrashed